.gitignore

```diff
@@ -2,3 +2,4 @@ Cargo.lock
 **/target
 **/*.rs.bk
 .rusk
+.idea
```

Comment:
We should figure out how to avoid such things – adding ignore for every IDE we could possibly use.
I'm not too familiar with IntelliJ, but maybe we can set it up to do not create the project folder in the current source file folder.

Reply:
This can be configured using a global excludes file on @miloszm machine. Here's a oneliner for you shell:

```sh
echo ".idea\n" >> ~/.gitignore && git config --global core.excludesFile "~/.gitignore"
```

Reply:
thanks Eduardo